MAINTAINERS.md

```diff
@@ -0,0 +1,18 @@
+# Test262 Maintainers Group
+
+Being part of the Test262 Maintainers Group comes with great power (merging things!) and the associated great responsibility (merging things!). Below is a list of the general responsibilities of Test262 maintainers and roughly how often the tasks are taken care of. Please remember, most maintainers do this in their free time or as one of many work tasks, and may therefore not respond as quickly as you hope.
+
+## Responsibilities
+- **Reviewing PRs** (1-2x/week)
+- **Reviewing test plans and helping guide contributors before PRs are submitted** (1-2x/week, as needed)
+- **Reviewing Stage 3 proposals and prioritizing related tests** (6x/yr, following TC39 plenaries)
+- **Fielding requests and suggestions for improvement from users(contributors, implementers, etc. TKTK link here to priority of constituencies TKTK)** (1x/month, at group meetings)
```

Comment:
I realize this implies once a month meetings instead of only after plenary. I think it can be changed to whatever we want.